homework-submission/homework-week1/app.js

```diff
@@ -0,0 +1,160 @@
+const Item = ({ id, description, deadline, done, removeItem }) => {
+  return (
+    <li key={id} className={done === true ? 'true' : ''}>
+      {description} {deadline} {done}
+      <span className="delete" onClick={() => removeItem(id)}>
+        DELETE
+      </span>
+    </li>
+  );
+};
+
+const TodoItems = ({ toDoList, removeItem }) => {
+  const length = toDoList.length;
+  const ListItems = length ? (
+    toDoList.map(({ id, done, description, deadline }) => {
+      return (
+        <Item
+          key={id}
+          id={id}
+          description={description}
+          deadline={deadline}
+          done={done}
+          removeItem={removeItem}
+        />
+      );
+    })
+  ) : (
+    <p>There is no items</p>
+  );
+
+  return <ul className="todo-items">{ListItems}</ul>;
+};
+
+class AddItems extends React.Component {
+  state = {
+    description: '',
+    deadline: '',
+  };
+
+  handelChange = e => {
+    this.setState({
+      [e.target.id]: e.target.value,
+    });
+  };
+
+  error() {
+    let element = document.getElementById('error');
+    element.classList.remove('hidde');
+  }
+
+  handelSubmit = e => {
+    e.preventDefault();
+    if (e.target['description'].value !== '' && e.target['deadline'].value !== '') {
```

Comment:
you are using two ways of handling values:
 - one by state
 - other using the form's target values

It would be shorter and easier to use one of them instead!